app/services/builders/chartkick/project_distribution_data_metrics/review_turnaround.rb

```diff
@@ -0,0 +1,18 @@
+module Builders
+  module Chartkick
+    module ProjectDistributionDataMetrics
+      class ReviewTurnaround
+        def retrieve_records(entity_id:, time_range:)
+          ::CompletedReviewTurnaround
+            .joins(review_request: :project)
+            .where(projects: { id: entity_id })
+            .where(created_at: time_range)
```

Comment:
don't we have other attribute more accurate than the `created_at` that's just the creation time of the record in our DB?

Reply:
I don't think we do.